web/src/nav/StatusMessagesNavItem.tsx

```diff
@@ -88,9 +108,31 @@ export class StatusMessagesNavItem extends React.PureComponent<Props, State> {
         this.subscriptions.unsubscribe()
     }
 
-    private renderMessage(message: GQL.IStatusMessage): JSX.Element | null {
-        switch (message.type) {
-            case GQL.StatusMessageType.CLONING:
+    private renderSyncErrorMessage(message: GQL.ISyncErrorStatusMessage): JSX.Element | null {
+        const displayName = message.externalServiceDisplayName
+        const extSvcID = message.externalServiceId
+
+        const title = `Syncing external service "${displayName}" failed:`
+        const linkTo = `/site-admin/external-services/${extSvcID}`
+        const linkText = `Edit "${displayName}"...`
```

Comment:
Personal preference: I don't think the `...` is necessary.

```suggestion
        const linkText = `Edit "${displayName}"`
```